test/bigtest/interactors/piece-details-modal.js

```diff
@@ -12,10 +12,17 @@ import Button from './button';
 }
 
 @interactor class BarcodeInput {
-  static defaultScope = '[class*=fieldWrapper] input[type="text"]';
+  static defaultScope = '[data-test-barcode]';
```

Comment:
I'm not sure if this is a correct way to use, what if there are several barcode inputs on the page?

Reply:
yes, it might be like that, but I don't see opportunity to determine unique identifier into MCL for particular field for the fields list